src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -1889,7 +1889,6 @@ public virtual void Dispose()
             if (_disposeOnceRunner.Disposed)
                 return;
 
-            GC.SuppressFinalize(this);
```

Comment:
No, You need the `SuppressFinalize` in the `_disposeOnceRunner` action, you create that in the ctor

Reply:
Done.